### PR TITLE
remove code duplication by loading configs and samples from the repo

### PR DESCRIPTION
**Pull request by matthidinger**
All files are loaded from the ~/samples and ~/schema dirs

1. Loaded the default card payload
2. Loaded the adaptive card schema into monaco
3. Moved all host app configs and loaded them from the samples dir

### CODE REVIEW

File: source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts [hunk 1]
```diff
@@ -1,904 +1,2 @@
...
-        "text",
-        "tel",
-        "url",
-        "email"
-      ]
-    }
-  }
-};
+import * as adaptiveCardSchemaJson from "../../../../schemas/adaptive-card.json";
+export const adaptiveCardSchema = adaptiveCardSchemaJson;
```
Comment (paulcam206) [inline source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts]: nice

**Comment by paulcam206** [comment]
Looks like there's a build failure, but not sure if it's related:

```
ERROR in ./src/app.ts
Module not found: Error: Can't resolve 'adaptivecards-designer' in 'C:\BA\1191\s\source\nodejs\adaptivecards-designer-app\src'
 @ ./src/app.ts 12:30-63

ERROR in C:\BA\1191\s\source\nodejs\adaptivecards-designer-app\src\app.ts
./src/app.ts
[tsl] ERROR in C:\BA\1191\s\source\nodejs\adaptivecards-designer-app\src\app.ts(3,29)
      TS2307: Cannot find module 'adaptivecards-designer'.
```

**Comment by matthidinger** [comment]
@dclaux since you're more familiar with the container styles changes are you able to resolve these conflicts and move the changes into the .json files? I also tried commenting out the clipboard code just to see if that was what caused the build failure. We will know more once the merge conflicts are gone

**Comment by dclaux** [comment]
Superseeded by https://github.com/Microsoft/AdaptiveCards/pull/2485

**Pull request closed without merging**